Unable to convert .jmx file to k6 using jmeter-to-k6, when using CSV dataset config element in my jmeter

Hello All,
I'm getting below error when I'm trying to convert my jmx file to k6. Its says Cannot find module 'tinyify'
I'm not sure why this issue is occuring. Please get it resolved soon. Using Jmeter version 5.0

(node:8518) UnhandledPromiseRejectionWarning: Error: Cannot find module 'tinyify' from '/Users/ashadrahman'
    at Function.module.exports [as sync] (/Users/ashadrahman/jmeter-to-k6/node_modules/resolve/lib/sync.js:71:15)
    at Browserify.plugin (/Users/ashadrahman/jmeter-to-k6/node_modules/browserify/index.js:388:29)
    at Promise (/Users/ashadrahman/jmeter-to-k6/src/build.js:83:13)
    at new Promise (<anonymous>)
    at bundle (/Users/ashadrahman/jmeter-to-k6/src/build.js:81:10)
    at buildCompat (/Users/ashadrahman/jmeter-to-k6/src/build.js:15:12)
(node:8518) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:8518) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

--- a/src/build.js
+++ b/src/build.js
@@ -1,6 +1,7 @@
 const browserify = require('browserify')
 const fs = require('fs')
 const ind = require('./ind')
+const tinyify = require('tinyify')
 const tmp = require('tmp')
 
 async function buildCompat (imports) {
@@ -80,7 +81,7 @@ async function stageIndex (dir, index) {
 async function bundle (dir) {
   return new Promise((resolve, reject) => {
     const bundler = browserify(`${dir}/index.js`, { standalone: 'Compat' })
-    bundler.plugin('tinyify', { flat: false })
+    bundler.plugin(tinyify, { flat: false })
     bundler.bundle((error, buffer) => {
       if (error) {
         reject(error)

